src/library_postprocessing.rs

```diff
@@ -505,4 +506,129 @@ impl Library {
             }
         }
     }
+
+    fn update_error_domain_functions(&mut self) {
+        // Find find all error domains that have corresponding functions
+        let mut error_domains = vec![];
+        for (ns_id, ns) in self.namespaces.iter().enumerate() {
+            'next_enum: for (id, type_) in ns.types.iter().enumerate() {
+                let type_ = type_.as_ref().unwrap(); //Always contains something
+                let enum_tid = TypeId {
+                    ns_id: ns_id as u16,
+                    id: id as u32,
+                };
+
+                if let Type::Enumeration(ref enum_) = *type_ {
+                    if let Some(ErrorDomain::Quark(ref domain)) = enum_.error_domain {
+                        let domain = domain.replace("-", "_");
+
+                        let mut function_candidates = vec![];
+                        function_candidates.push(domain.clone());
+                        if !domain.ends_with("_quark") {
+                            function_candidates.push(format!("{}_quark", domain));
+                        }
+                        if !domain.ends_with("_error_quark") {
+                            if domain.ends_with("_quark") {
+                                function_candidates
+                                    .push(format!("{}_error_quark", &domain[..(domain.len() - 6)]));
+                            } else {
+                                function_candidates.push(format!("{}_error_quark", domain));
+                            }
+                        }
+                        if let Some(domain) = domain.strip_suffix("_error_quark") {
+                            function_candidates.push(domain.to_owned());
+                        }
+                        if let Some(domain) = domain.strip_suffix("_quark") {
+                            function_candidates.push(domain.to_owned());
+                        }
+
+                        if let Some(func) = ns.functions.iter().find(|f| {
+                            function_candidates
+                                .iter()
+                                .any(|c| f.c_identifier.as_ref() == Some(c))
+                        }) {
+                            error_domains.push((
+                                ns_id,
+                                enum_tid,
+                                None,
+                                func.c_identifier.as_ref().unwrap().clone(),
+                            ));
+                            continue 'next_enum;
```

Comment:
Why not just `continue` btw?

Reply:
Here it could be plain `continue` but I thought it would be less confusing to have it with a label in both cases so it's clear that both `continue`s do the same and that there's not another loop hidden somewhere :)

Reply:
Fine by me!